hydra-node/src/Hydra/Chain/Direct/Tx.hs

```diff
@@ -390,19 +390,18 @@ fanoutTx ::
   UTxO ->
   -- | Everything needed to spend the Head state-machine output.
   UTxOWithScript ->
-  -- | Point in time at which this transaction is posted, used to set
-  -- lower bound.
-  PointInTime ->
+  -- | Contestation deadline as SlotNo, used to set lower tx validity bound.
+  SlotNo ->
   -- | Minting Policy script, made from initial seed
   PlutusScript ->
   Tx
-fanoutTx utxo (headInput, headOutput, ScriptDatumForTxIn -> headDatumBefore) (slotNo, _) headTokenScript =
+fanoutTx utxo (headInput, headOutput, ScriptDatumForTxIn -> headDatumBefore) deadlineSlotNo headTokenScript =
   unsafeBuildTransaction $
     emptyTxBody
       & addInputs [(headInput, headWitness)]
       & addOutputs orderedTxOutsToFanout
       & burnTokens headTokenScript Burn headTokens
-      & setValidityLowerBound slotNo
+      & setValidityLowerBound (deadlineSlotNo + 1)
```

Comment:
would be nice to elaborate more on this +1.
why is this more robust?
can we give an example?